com.dropbox.Client.appdata.xml

```diff
@@ -0,0 +1,70 @@
+<description xml:lang="ro"><p>Ai nevoie sa iti stochezi documente importante,imagini,muzica,si fisiere?Foarte simplu!Cu aceasta aplicatie poti folosi internetul pentru a salva tot ce este important pentru tine.Poti accesa aceste fisiere de pe orice computer care are conexiune la internet.Foloseste acest serviciu pentru a face o copie de rezerva a fisierelor,impartasi imagini sau colaboreaza cu un proiect doar impartasind un folder din Dropbox cu oricine vrei.Creeaza un cont gratuit si incepe sa salvezi si sa impartasesti chiar de azi! ** Necesita conexiune la internet.</p></description>
+<description xml:lang="th"><p>ต้องการที่จะเก็บเอกสารสำคัญ ภาพถ่าย เพลงและไฟล์ของคุณอย่างปลอดภัยหรือไม่ ง่ายๆ! ด้วยแอพนี้คุณสามารถใช้อินเทอร์เน็ตในการเก็บทุกอย่างที่สำคัญกับคุณ จากนั้นคุณสามารถเข้าถึงไฟล์จากคอมพิวเตอร์ เครื่องใดก็ได้ที่เชื่อมต่ออินเทอร์เน็ต ใช้บริการนี้เพื่อสำรองไฟล์ของคุณ แบ่งปันภาพถ่ายหรือทำงานร่วมกันในโครงการโดยการแชร์โฟลเดอร์ Dropbox กับคนที่คุณต้องการ สร้างบัญชีฟรีและเริ่มต้นการเก็บและแชร์มันได้เลยวันนี้! ** ต้องใช้อินเทอร์เน็ต.</p></description>
+<description xml:lang="vi"><p>Bạn cần lưu trữ an toàn tất cả các văn bản, hình ảnh, bài hát và các tập tin quan trọng? Dễ dàng thôi! Với ứng dụng này, bạn có thể sử dụng Internet để lưu tất cả mọi thứ quan trọng với bạn. Sau đó, bạn có thể truy cập các tệp này từ bất kỳ máy tính nào có kết nối internet. Sử dụng dịch vụ này để sao lưu các tập tin, chia sẻ hình ảnh hoặc cộng tác trên một dự án bằng cách chia sẻ thư mục Dropbox với bất cứ ai mà bạn muốn. Hãy tạo tài khoản miễn phí và bắt đầu lưu và chia sẻ ngày hôm nay! ** Yêu cầu kết nối Internet.</p></description>
+<description xml:lang="zh_CN"><p>想要安全地存储所有重要文件、照片、音乐和文件吗？别急！只要有了这个app，您就可以使用互联网来保存所有重要文件。您还可以从任何有网络连接的计算机上访问这些文件。使用该服务来备份文档，通过共享一个Dorpbox文件夹来共享照片，合作项目。创建一个免费账户，开始保存和分享您的每一天吧！.</p></description>
+<description xml:lang="zh_TW"><p>需要安全地儲存你的所有文件、照片、歌曲和檔案嗎？很簡單！使用這款應用程式，你可以使用網路來儲存所有你認為重要的東西。然後你可以從任何一台擁有網路連線的電腦上存取那些檔案。和你想要分享的人一起共享 Dropbox 資料夾，使用這個服務來備份檔案、分享照片或在同一個專案上進行合作。今天就建立免費帳號，並開始儲存和分享吧！ ** 需要網路連線.</p></description>
+<description><p>Need to safely store all of your important documents, photos, songs, and files? Easy! With this app you can use the internet to save everything that’s important to you. You can then access those files from any computer that has an internet connection. Use this service to backup your files, share photos or collaborate on a project by sharing a Dropbox folder with whomever you’d like. Create a free account and start saving and sharing today!  ** Requires internet.</p></description>
```

Comment:
Please add the text:
```    
<p>NOTE: This wrapper is not verified by, affiliated with, or
    supported by Dropbox, Inc.</p>
```
If you want to translate it into Spanish too that'd be ace ;-) I won't ask you to do any of the others yourself.

Reply:
In Spanish:

```
<p>NOTE: Este paquete de instalación no es soportado ni verificado por la compañía <strong>Dropbox, Inc.</strong>; asimismo, tampoco tiene algún tipo asociación con la misma.</p>
```

Reply:
Thanks! Any other languages are more than welcome.

Reply:
> Please add the text:
> 
> <p>NOTE: This wrapper is not verified by, affiliated with, or
>     supported by Dropbox, Inc.</p>

@nedrichards I would replace `wrapper` with `application`, if you don't mind. Users should not need to understand what "wrapper" is to understand that note, which can become confusing otherwise.

> In Spanish:
> 
> <p>NOTA: Este paquete de instalación no es soportado ni verificado por la compañía <strong>Dropbox, Inc.</strong>; asimismo, tampoco tiene algún tipo de asociación con la misma.</p>

@x80486 Thanks! You did beat me to it :-)